capstone/src/main/java/com/google/step/servlets/PageViewServlet.java

```diff
@@ -0,0 +1,94 @@
+package com.google.step.servlets;
+
+import com.google.gson.Gson;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import java.util.List;
+
+import com.google.step.clients.MetricsClient;
+import com.google.step.data.WeeklyPageView;
+import com.google.step.data.RestaurantPageViews;
+
+/**
+ * Servlet that will read and write page view metrics to and from a persistent store.
+ */
+@WebServlet("/page-view")
+public class PageViewServlet extends HttpServlet {
+
+    private final MetricsClient metricsClient = new MetricsClient();
+    private static Gson gson = new Gson();
+
+    /**
+     * Method that retrieves different page view metrics depending on the query parameters.
+     * @param request with a mix of the following parameters: <String, restaurantKey> and <int, year>.
+     *                These can be in the request or not at all, and depending on the mix a different
+     *                method from the client will be called.
+     *                String restaurantKey -> getCurrentPageViews()
+     *                int year, String restaurantKey -> getYearRestaurantPageViews()
+     *                empty -> getAllPageViews();
+     *                A SC_FORBIDDEN status will be set if none of these three specified combinations
```

Comment:
Update this comment (no longer returns forbidden)

Reply:
Here and below

Reply:
Fixed.